src/embedded/embeddedClickEvents.ts

```diff
@@ -1,30 +1,33 @@
 import {
-  EmbeddedManager,
-  EmbeddedMessageElementsButton,
+  IterableEmbeddedManager,
+  IterableEmbeddedButton,
   IterableEmbeddedMessage
 } from '../embedded';
 
-const embeddedManager = new EmbeddedManager();
-export const handleElementClick = (message: IterableEmbeddedMessage) => {
+export const handleElementClick = (
+  embeddedManager: IterableEmbeddedManager,
```

Comment:
@mprew97 Should we be exporting the things in this file? I think these should only be used under the hood by the OOTB classes, not by customers...

Reply:
they need to be exported by this file. We can lock them down from being exported outside of the SDK as a whole though, if that makes sense. Great callout, will go ahead and do that.

Reply:
Just verified, these aren't being exported by the SDK as it stands.